app/components/candidate_interface/referee_guidance_component.rb

```diff
@@ -10,6 +10,13 @@ def pluralize_provider
       'provider'.pluralize(provider_count)
     end
 
+    def reject_by_default_days
+      @reject_by_default_days ||= TimeLimitCalculator.new(
+        rule: :reject_by_default,
+        effective_date: @application_form&.application_choices&.first&.sent_to_provider_at || Time.zone.now,
```

Comment:
We trust that `application_form.application_choices` are present (even if empty) in `provider_count` below — could we do away with the first two `&.`?